tests/cypress/src/DashTable.ts

```diff
@@ -1,26 +1,47 @@
+const EDITABLE = `.dash-spreadsheet:not(.dash-loading)`;
+const NOT_EDITABLE = `.dash-spreadsheet.dash-loading`;
+const ANY = `.dash-spreadsheet`;
+
+export enum State {
+    Editable,
+    NotEditable,
+    Any
+}
+
+const getSelector = (state: State) => {
+    switch (state) {
+        case State.Editable:
+            return EDITABLE;
+        case State.NotEditable:
+            return NOT_EDITABLE;
+        default:
+            return ANY;
+    }
+};
+
 export default class DashTable {
-    static getCell(row: number, column: number) {
-        return cy.get(`#table tbody tr td.column-${column}`).eq(row);
+    static getCell(row: number, column: number, editable: State = State.Editable) {
+        return cy.get(`#table ${getSelector(editable)} tbody tr td.column-${column}`).eq(row);
```

Comment:
With the loading state that affects the table's rendering and behavior it's now important to be able to select the cells in the correct state (or any state) based on what's tested. Default is `editable` for backward compatibility with the tests.